alias/gui.c

```diff
@@ -53,11 +53,33 @@ int alias_sort_name(const void *a, const void *b)
   const struct AliasView *av_a = a;
```

Comment:
As this is a standard `sort` callback function, we can be brief in the Doxygen comment.
Add ` - Implements ::sort_t` to the description, and drop all the `@param`s and `@retval`s.

Add a `@note` mentioning that non-visible Aliases will be sorted to the end